io.asgardeo.tomcat.oidc.agent/src/main/java/io/asgardeo/tomcat/oidc/agent/OIDCAgentFilter.java

```diff
@@ -132,6 +136,29 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
         }
     }
 
+    private String resolveTargetPage(RequestContext requestContext) {
```

Comment:
Here the priority should be given to REDIRECT_URI_KEY right?
if the value is not present only we should set the target page as the home page.